### DISCUSSION TITLE
Stop cypress from automatic updating new version

### DISCUSSION THREAD

**Discussion by tsaqif-yusof** [cypress-io/cypress#14732] [Questions and Help] [answered]
hi. i new with cypress. i want to ask on how to stop cypress from automatic updating new version every time when start with "npx cypress open"

![Screenshot 2021-01-25 at 8 10 52 PM](https://user-images.githubusercontent.com/77956042/105704231-7380bc00-5f49-11eb-91a1-c9d42f591a95.png)

please. because everytime i open cypress. it will update if has new version. because my laptop slow. sometime i need to "npx cypress open" twice when it failed for first time. this take time for my side to open the cypress.

**Comment by bahmutov** [comment, marked as answer]
Install Cypress as a local dev dependency first in your project with `npm i -D cypress`. Then you can use `npx cypress open` and it will use the locally installed Cypress. Without installing Cypress your command `npx cypress open` downloads and installs Cypress, opens it, but when you exit `npx` removes the install. So the next time you run it, it does it again.

**Reply by tsaqif-yusof** [reply]
hi @bahmutov . thanks for the reply. what you mean when i exit `npx` removes the install? does it mean when i close my own computer. then tomorrow i need to use cypress. i need to do `npm i -D cypress` before `npx cypress open` ?

**Reply by fornever847** [reply]
How?

**Comment by bahmutov** [comment]
Yes you have to do npm i -D cypress. That will add cypress to the dev dependencies in the package json file. Then whenever one does npm install it will install cypress. And then you can do npx cypress open to just open the preinstalled cypress.

**Reply by tsaqif-yusof** [reply]
i see. thank you. i will try it.

**Reply by djangofan** [reply]
yeah, but it will still prompt you asking to update right?
